_docs/welcome/notes/0.4.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Istio 0.4
+
+order: 97
+
+layout: docs
+type: markdown
+---
+
+## General
+
+This release has only got a few weeks worth of changes, as we stabilize our monthly release process. 
+In addition to the usual pile of bug fixes and perf improvements, this release includes:
+
+- **Cloud Foundry**. Added support for the [Cloud Foundry](https://www.cloudfoundry.org) platform to Pilot, making it
```

Comment:
Possibly rephrase to

Minimum Pilot support for [Cloud Foundry](https://www.cloudfoundry.org) that enables discovery of CF services and service instances. 

The additional line "This represents the strict minimum" is fairly wordy and should be collapsed into the first sentence or it should highlight the feature stability (Experimental,Alpha, etc.)

Reply:
OK.